Content/src/Client/Client.fs

```diff
@@ -88,6 +94,9 @@ module Server =
       |> Remoting.buildProxy<ICounterApi>
     #endif
 let initialCounter = Server.api.initialCounter
+#elseif (bridge)
```

Comment:
this can be removed?

Reply:
This one is preventing the else branch

Reply:
I see, let's change the `else` branch to `elseif (!bridge)` instead - those empty lines are misleading

Reply:
Nice solution! Will do